### ISSUE TITLE
Font install wont install at windows (Emacs installed by scoop)

### ISSUE THREAD

**Issue by xz-dev** [doomemacs/doomemacs#7596]
### I confirm that...

- [X] I have searched the [issue tracker](https://github.com/doomemacs/doomemacs/issues), [documentation](https://docs.doomemacs.org), [FAQ](https://docs.doomemacs.org/-/faq), [Discourse](https://discourse.doomemacs.org), and [Google](https://google.com), in case this issue has already been reported/resolved.

- [X] I have read "[How to Debug Issues](https://doomemacs.org/d/how2debug)", and will use it to provide as much information about this issue as possible.

- [X] The issue can be reproduced on the **latest** available commit of Doom.

- [X] The issue can be reproduced on a stable release of Emacs, such as 27, 28, or 29. *(Unstable versions end in .50, .60, or .9x)*

### Expected behavior

Use `all-the-icons-install-fonts` and `nerd-icons-install-fonts` to try to install font for fix https://github.com/doomemacs/doomemacs/issues/7522
But not work

### Current behavior

It work at Linux but not in Windows,

### Steps to reproduce

1. Open Emacs
2. type nerd-icons-install-fonts
3. Choice ~/AppData/Local/Microsoft/Windows/Fonts
4. Install finish
5. Restart Emacs
6. No icon show
7. Check Windows Font choicer, no icon font

### System Information

https://gist.github.com/MS-Xiangzhe/b721ad42b9beba05abb0b25b696a529d

**Comment by Schievel1** [comment]
Yes, in windows it is not as easy as just shoving the fonts into a directory. Install them instead like this:
In windows, start emacs and do `M-x nerd-icons-install-fonts`. It will ask you if you really want to do this, so answer yes and then it will ask you for the directory. Just use your homefolder. Then open windows explorer and go to that directory. The file it downloaded will be named NFM.ttf. Open it. In the font viewer windows that opens up, click install. Then restart emacs. 

There is no `all-the-icons-install-fonts` anymore.

**Comment by HarryTheFairy** [comment]
Thanks Shievel1. Tried to fix this for hours now, but your comment finally solved it. nerd-icons-install-fonts is not enough. I also tried to download and install fonts from https://www.nerdfonts.com/ but althoug it was selectable in menu-set-font the font did not change. Not sure if i am missing an important step for using menu-set-font of if this is just not meant to work on windows.

One more hint, I think after installing the NFM.ttf i also needed to restart my machine. Reloading or restarting emacs alone was not enough in my case.
